src/java/map/BinaryMask.java

```diff
@@ -167,6 +167,21 @@ public BinaryMask randomWalkWithBrush(Vector2f start, String brushName, int size
         return this;
     }
 
+    public BinaryMask guidedWalkWithBrush(Vector2f start, Vector2f target, String brushName, int size, int numberOfUses,
```

Comment:
All the other connect with brush functions should use this to reduce code duplication

Reply:
It's not the same thing though.  This one is guided, while other connect with brush functions are not guided like this.  There are times when non-guided connect functions are better to use.